.github/actions/build-e2e-matrix/action.yml

```diff
@@ -50,6 +50,7 @@ runs:
             ["visualizations-tabular", {} ],
             ["oss-subset", { edition: 'oss', context: "special" } ],
             ["slow", { runner: beefierRunner, context: "special" } ],
+            ["flaky", { runner: beefierRunner, context: "special" } ],
```

Comment:
What about flakes that can run using only OSS uberjar?

Reply:
I just counted, and there are only 12 such tests in our entire test suite. I think it's best not to handle this.